.doctrine-project.json

```diff
@@ -5,21 +5,26 @@
     "docsSlug": "doctrine-persistence",
     "versions": [
         {
-            "name": "3.1",
-            "branchName": "3.1.x",
+            "name": "3.2",
+            "branchName": "3.2.x",
             "slug": "latest",
             "upcoming": true
         },
         {
-            "name": "3.0",
-            "branchName": "3.0.x",
-            "slug": "3.0",
+            "name": "3.1",
+            "branchName": "3.1.x",
```

Comment:
```suggestion
            "branchName": "3.1.x",
            "slug": "3.1",
```

Reply:
:facepalm: I need rest after this symfonycon :sweat:

Reply:
I need to create a schema for this sometime that will be checked in a workflow.